test_package/conanfile.py

```diff
@@ -28,19 +29,28 @@ def imports(self):
         embedded_python_tools.symlink_import(self, dst="bin/python")
         self.copy("licenses/*", dst="licenses", folder=True, ignore_case=True, keep_path=False)
 
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
     def _test_env(self):
         """Ensure that Python runs and finds the installed environment"""
         script = "import sys; print(sys.version);"
         name = str(self.options.env)
         if self.options.env:
             script += f"import {name}; print('Found {name}');"
 
-        self.run(f".\\bin\\python\\python.exe -c \"{script}\"")
+        python_exe = pathlib.Path("./bin/python/python").resolve()
+        self.run(f"{python_exe} -c \"{script}\"")
```

Comment:
Would it not be better to let conan handle the quoting / escaping? This also goes for the other `run` calls in this PR. 

```suggestion
        self.run([f"{python_exe}",  "-c", f"{script}"])
```

Reply:
Indeed, that's much better! I didn't know it accepted a list of arguments as well. That's great. (Looks like it's implemented in terms of `subprocess.run()`.)